sonar-scanner-engine/src/main/java/org/sonar/scanner/sensor/DefaultSensorStorage.java

```diff
@@ -372,13 +381,11 @@ public void store(DefaultSymbolTable symbolTable) {
     if (writer.hasComponentData(FileStructure.Domain.SYMBOLS, componentRef)) {
       throw new UnsupportedOperationException("Trying to save symbol table twice for the same file is not supported: " + symbolTable.inputFile().absolutePath());
     }
+    final ScannerReport.Symbol.Builder builder = ScannerReport.Symbol.newBuilder();
+    final ScannerReport.TextRange.Builder rangeBuilder = ScannerReport.TextRange.newBuilder();
     writer.writeComponentSymbols(componentRef,
-      Iterables.transform(symbolTable.getReferencesBySymbol().entrySet(), new Function<Map.Entry<TextRange, Set<TextRange>>, ScannerReport.Symbol>() {
-        private ScannerReport.Symbol.Builder builder = ScannerReport.Symbol.newBuilder();
-        private ScannerReport.TextRange.Builder rangeBuilder = ScannerReport.TextRange.newBuilder();
-
-        @Override
-        public ScannerReport.Symbol apply(Map.Entry<TextRange, Set<TextRange>> input) {
+      symbolTable.getReferencesBySymbol().entrySet().stream()
+        .map(input -> {
           builder.clear();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 9. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)